### ISSUE TITLE
Write an ingestion script that can accept an explicit issue date

### ISSUE THREAD

**Issue by krivard** [cmu-delphi/delphi-epidata#224]
If we have historical data versioning information and we want to include it in a new signal, currently there's no way to do that, since ingestion defaults to an issue date of $today.

We essentially need either a fork or non-default handling that pieces together [csv_to_database.py](https://github.com/cmu-delphi/delphi-epidata/blob/main/src/acquisition/covidcast/csv_to_database.py) with the following changes:
1. an alternate upload path
2. some kind of directory structure that encodes each issue
3. setting the issue date explicitly when it calls [find_csv_files()](https://github.com/cmu-delphi/delphi-epidata/blob/204f35e7e0cbbc4971c45a48631ce5c4b6b8ac35/src/acquisition/covidcast/csv_importer.py#L79)

**Comment by melange396** [comment]
https://github.com/cmu-delphi/delphi-epidata/pull/230

**Comment by melange396** [comment]
in practice, youll probably want to call this with something like:
`python3 csv_to_database.py -specific_issue_date --data_dir=/common/maria/claims_hosp_issues/ --is_wip_override`

but in testing, i ran:
`docker run --rm -t -i --mount type=bind,source="${HOME}",target=/data_dir --network delphi-net delphi_python_mine python3`
and then pasted into the interactive python shell:
```
from delphi.operations import secrets
secrets.db.host = 'delphi_database_epidata'
secrets.db.epi = ('user', 'pass')
import delphi.epidata.acquisition.covidcast.csv_to_database as csvdb

args = csvdb.get_argument_parser().parse_args(['--specific_issue_date', '--data_dir=/data_dir/t', '--is_wip_override'])
csvdb.main(args)
```

**Comment by krivard** [comment]
Merged cmu-delphi/delphi-epidata#230
